tick/inference/hawkes_conditional_law.py

```diff
@@ -430,7 +445,7 @@ def _lexical_to_index(self, i, j, l):
         """
         return self._ijl2index[i][j][l]
 
-    def incremental_fit(self, realization, compute=True):
+    def incremental_fit(self, realization, T=None, compute=False):
```

Comment:
I truly think `incremental_fit` should have `compute=True` by default. Default behavior of `incremental_fit` should be that model is ready after calling it. This is what has been done with other learners.